pkg/block/azure/adapter.go

```diff
@@ -605,3 +631,20 @@ func (a *Adapter) GetPresignUploadPartURL(_ context.Context, _ block.ObjectPoint
 func (a *Adapter) ListParts(_ context.Context, _ block.ObjectPointer, _ string, _ block.ListPartsOpts) (*block.ListPartsResponse, error) {
 	return nil, block.ErrOperationNotSupported
 }
+
+func ParseURL(raw string) (string, string, error) {
```

Comment:
Let's add documentation and named args.. `(string, string, error)` is not very indicative.